src/early_multicellular_stage/editor/EarlyMulticellularEditor.cs

```diff
@@ -435,14 +435,14 @@ private void OnStartEditingCellType(string name, bool switchTab)
             return;
         }
 
-        var newTypeToEdit = EditedSpecies.CellTypes.First(c => c.TypeName == name);
+        var newTypeToEdit = EditedSpecies.CellTypes.FirstOrDefault(c => c.TypeName == name);
 
         // Only reinitialize the editor when required
         if (selectedCellTypeToEdit == null || selectedCellTypeToEdit != newTypeToEdit)
```

Comment:
The check against null edited cell type, doesn't work as intended here anymore. I think the intention was to force initialize the editing if the previous type to be selected was null (which would mean that now this is used), but with being able to call this with a null type now, this code will always trigger even if the previous type was null and the new type was null.

Reply:
Yeah, that's a good point